locator/topology.cc

```diff
@@ -159,7 +159,9 @@ const node* topology::add_node(node_holder nptr) {
     nptr->set_idx(_nodes.size());
     _nodes.emplace_back(std::move(nptr));
 
-    tlogger.debug("topology[{}]: add_node: {}, at {}", fmt::ptr(this), debug_format(node), current_backtrace());
+    if (tlogger.is_enabled(log_level::debug)) {
+        tlogger.debug("topology[{}]: add_node: {}, at {}", fmt::ptr(this), debug_format(node), current_backtrace());
+    }
```

Comment:
looks good. we could also have implemented a lazy backtrace, which invokes current_backtrace() in the formatter. But maybe not worth the effort.

Reply:
agreed.  but not worth the effort